Testscripts/Linux/channel_change.sh

```diff
@@ -36,12 +34,33 @@ prefix1="Rel_ID="
 suffix1=","
 prefix2="target_cpu="
 
-function reset_cpu_id() {
-# Read VMBUS ID and store vmbus ids in the array
-# Change the cpu_id from non-zero to 0 in order to set cpu offline.
-# If successful, all cpu_ids in lsvmbus output should be 0 per channel, and generate
-# the list of idle cpus
+function Main() {
+	FailedCount=0
+	# Collect the vmbus and cpu id information from the system, and store in /tmp/lsvmbus.output
+	basedir=$(pwd)
+	lsvmbus
+	if [ $? != 0 ]; then
+		if [ -f $basedir/linux/tools/hv/lsvmbus ]; then
+			chmod +x $basedir/linux/tools/hv/lsvmbus
+			$basedir/linux/tools/hv/lsvmbus -vv > $lsvmbus_output_location
+		else
+			LogErr "File, lsvmbus, not found in the system. Aborted the execution."
+			SetTestStateAborted
+			exit 0
+		fi
+	else
+		lsvmbus -vv > $lsvmbus_output_location
+	fi
+	LogMsg "Successfully recorded lsvmbus output in $lsvmbus_output_location"
 
+	# Before chaning cpu id to 0, reset all cpu ids of each vmbus channel.
```

Comment:
changing

Reply:
Fixed in 56e05a9ae33d54c5d10114fd7fafcc8ac7eb5f83